Automattic/jetpack
# Jetpack Plugin: Unify money-back guarantee component

### Introduction



This PR aims to unify the look and feel of "Money back guarantee" indicators going from the Jetpack plugin to jetpack.com (checkout).

---

### Changes proposed in this Pull Request:

* Change the look of `<MoneyBackGuarantee>` to create better consistency when going to checkout.

---

### Jetpack product discussion



* p1HpG7-cY7-p2#comment-48894

---

### Does this pull request change what data or activity we track or use?



Nope!

---

### Testing instructions:






* Spin up the Jetpack plugin with this branch
* Create a Jetpack Free connection
* Go to `/wp-admin/admin.php?page=jetpack#/recommendations/product-suggestions`
* Verify that the <MoneyBackGuarantee> component looks like the **Updated look** screenshot.

---

### Screenshots

**Updated look:**

![Screenshot 2021-10-01 at 16 24 09](https://user-images.githubusercontent.com/3846700/135636409-d086733e-650f-46ea-8423-f6b68647bca3.png)

**Existing look:**

![Screenshot 2021-10-01 at 16 21 28](https://user-images.githubusercontent.com/3846700/135636344-012a383c-ab27-4922-8194-46f44be42e6a.png)
